micro-ui/web/micro-ui-internals/packages/libraries/src/hooks/pgr/useInboxData.js

```
@@ -47,6 +47,18 @@ const mapWfBybusinessId = (wfs) => {
 
 const combineResponses = (complaintDetailsResponse, workflowInstances) => {
   let wfMap = mapWfBybusinessId(workflowInstances.ProcessInstances);
+  let filtered=complaintDetailsResponse?.ServiceWrappers?.filter((complaint) => wfMap?.[complaint?.service?.serviceRequestId])
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'filtered' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)